app/views/orders/index.html.erb

```diff
@@ -17,6 +34,13 @@
         <td><%= order.reward_snapshot.title %></td>
         <td><%= order.reward_snapshot.description %></td>
         <td><%= order.reward_snapshot.price %></td>
+        <td>
+          <% if order.delivered %>
```

Comment:
that's fine, but I'd use I18n to do that. Google how you could localize a boolean